### DISCUSSION TITLE
Issue with Plausible on Digital Ocean

### DISCUSSION THREAD

**Discussion by mrdshyml** [plausible/analytics#2831] [Self-Hosted Support]
I am trying to setup my self-hosted instance of plausible on a digital ocean droplet using this guide here https://www.digitalocean.com/community/tutorials/how-to-install-plausible-analytics-on-ubuntu-22-04

While I have completed all the steps, I am not being able to access my plausible instance using the domain name I have mapped by IP address to. I am getting a `502 Bad Gateway.`

I have rechecked all the steps and everything is fine except when I try to `curl http://localhost:8000` and I get this error `curl: (52) Empty reply from server`.

How can I resolve this issue?

**Comment by ruslandoga** [comment]
👋 @mrdshyml 

Can you please post your `docker-compose.yml`, `plausible-conf.env` (without secret info) and other configuration you've done here?

**Reply by mrdshyml** [reply]
Here is my `docker-compose.yml` :
<img width="950" alt="Screenshot 2023-04-07 103605" src="https://user-images.githubusercontent.com/80547432/230544942-47a9993f-dd55-4590-899d-7d2dd0e90aa4.png">
Here is my `plausible-conf.env`. I am bit confused, I did not see any admin email etc on this file. 
```
BASE_URL=plausible.mydomain.in
SECRET_KEY_BASE= {my base64 key}
```

**Reply by ruslandoga** [reply]
Have you setup any reverse proxying?

**Reply by ruslandoga** [reply]
What do logs show? `docker compose logs`

**Reply by mrdshyml** [reply]
I am a bit new to tech as a whole. So i find it hard to understand some jargon. I have completed everything up to step 3 Installing certbot 
 in this article. https://www.digitalocean.com/community/tutorials/how-to-install-plausible-analytics-on-ubuntu-22-04#step-2-installing-and-configuring-nginx

**Reply by ruslandoga** [reply]
Sure, no problem. Reverse proxying is the NGINX step in the article.

**Reply by ruslandoga** [reply]
I think executing `docker compose logs` in your `hosting` directory (where all the Plausible files and `docker-compose.yml` are located) would provide us with the information necessary to debug the problem :)

**Reply by mrdshyml** [reply]
```
plausible-plausible-1            | Crash dump is being written to: erl_crash.dump...done
plausible-plausible-1            | ERROR! Config provider Config.Reader failed with:
plausible-plausible-1            | ** (RuntimeError) BASE_URL must start with `http` or `https`. Currently configured as `plausible.mydomain.in`
```
This is what I have got in logs. I have update base URL to include `http://`. But the issue is not resolved yet.

**Reply by ruslandoga** [reply]
What do logs show now?

**Reply by mrdshyml** [reply]
Well, i think changing the `base url` resolved the issue for me. I took a few minutes. Everything is working well for me. Thanks for guidance.
